backend/tests/v5/unit/middleware/sso/aad.test.js

```diff
@@ -261,7 +263,9 @@ const testAuthenticate = () => {
 
 			await Aad.authenticate(redirectUri)(req, res);
 			expect(Sso.addPkceProtection).toHaveBeenCalledTimes(1);
-			expect(Responder.respond).not.toHaveBeenCalled();
+			expect(Responder.respond).toHaveBeenCalledTimes(1);
+			expect(Responder.respond).toHaveBeenCalledWith(req, res, templates.ok, expect.anything());
```

Comment:
could we maybe use

https://jestjs.io/docs/expect#expectobjectcontainingobject

Reply:
thought we couldn't but i guess `expect.any()` inside the link property would work here! 👍